lib/widgets/myTile.dart

```diff
@@ -16,7 +16,7 @@ class MyTile extends StatelessWidget {
   Widget build(BuildContext context) {
     return ListTile(
       title: wTitle ?? Text(title),
-      subtitle: wSubTitle ?? Text(subTitle),
+      subtitle: wSubTitle ?? subTitle == null ? SizedBox() : Text(subTitle),
```

Comment:
why this

Reply:
data must not be null issue arises if subtitle's value is absent while passing

Reply:
rather use default value as empty string

Reply:
`subtitle: Text(wSubTitle ?? "") `
Is this will work?

Reply:
> `subtitle: Text(wSubTitle ?? "") `
> Is this will work?

`subtitle: wSubTitle ?? Text(subTitle ?? "")
`